packages/theme-patternfly-org/components/tableOfContents/tableOfContents.js

```diff
@@ -52,17 +52,17 @@ const ReleaseNotesTOC = () => (
   </Grid>
 );
 
-const renderItem = item => {
+const renderItem = (item, index) => {
   if (Array.isArray(item)) {
     return (
-      <ul key={item.join('-')}>
+      <ul key={index}>
         {item.map(renderItem)}
       </ul>
     );
   }
 
   return (
```

Comment:
When i click on a ToC link it doesn't scroll the item into view. Same when you try and load the page to an item directly, i.e. `https://patternfly-org-pr-2026-v4.surge.sh/v4/components/drawer#breakpoint`

Reply:
Good catch! I'm not sure why it's not scrolling... will investigate.

Reply:
The reason it's not scrolling is that surge sometimes postpends `/` to URLs... https://patternfly-org-pr-2026-v4.surge.sh/v4/components/about-modal/#custom-background-image does NOT work, but https://patternfly-org-pr-2026-v4.surge.sh/v4/components/about-modal#custom-background-image does.

Reply:
no matter what I do, a slash is added before the hashtag, even when clicking on the second link here. Or when the page is already open and I delete the slash and hit Enter it is also re-inserted

Reply:
I _think_ this is mostly fixed. I'd love steps to reproduce if you can.